symphony/content/content.blueprintsdatasources.php

```diff
@@ -168,7 +168,7 @@ public function __form()
 
                             // Handle pre Symphony 2.2.1 Static DS's
                         } else {
-                            $fields['static_xml'] = trim($existing->grab());
+                            $fields['static_xml'] = trim($existing->execute());
```

Comment:
;) Thanks mate. Using Scrutinizer bugs as a start for @ChoboHub to learn system has been great so far tho.